hypothesis-python/tests/cover/test_database_backend.py

```diff
@@ -62,6 +63,12 @@
 from tests.common.utils import checks_deprecated_behaviour, skipif_emscripten
 from tests.conjecture.common import nodes, nodes_inline
 
+if sys.platform == "emscripten":
+    # possibly FIXME?
+    pytest.skip(
+        "unsupported threads created during collection", allow_module_level=True
+    )
+
```

Comment:
very likely from the background write database in the new `test_database_not_equal` test: `(BackgroundWriteDatabase(InMemoryExampleDatabase()), InMemoryExampleDatabase())`. We could skip that param on emscripten, or defer db creation from collection-time to test-time with a lambda.

(and maybe `BackgroundWriteDatabase` should not create and start a thread until its first `fetch`/`delete`/`save` operation?)

Reply:
I think deferring thread creation is a good idea.  Also happy to skip on emscripten, since it doesn't (yet?) have threads.

Reply:
Yup. I implemented deferred thread creation as suggested, and will skip whatever is necessary once CI results are in.